app/controllers/api/teams.php

```diff
@@ -304,7 +306,9 @@
     ->label('sdk.response.model', Response::MODEL_MEMBERSHIP)
     ->label('abuse-limit', 10)
     ->param('teamId', '', new UID(), 'Team ID.')
-    ->param('email', '', new Email(), 'Email of the new team member.')
+    ->param('userId', '', new UID(), 'User ID.', true)
```

Comment:
Let's keep `email` first to prevent breaking changes on SDK side.

Reply:
done!